contracts/src/v0.8/l2ep/dev/scroll/ScrollValidator.sol

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.19;
+
+import {AggregatorValidatorInterface} from "../../../shared/interfaces/AggregatorValidatorInterface.sol";
+import {TypeAndVersionInterface} from "../../../interfaces/TypeAndVersionInterface.sol";
+import {ScrollSequencerUptimeFeedInterface} from "../interfaces/ScrollSequencerUptimeFeedInterface.sol";
+
+import {SimpleWriteAccessController} from "../../../shared/access/SimpleWriteAccessController.sol";
+
+import {IL1ScrollMessenger} from "@scroll-tech/contracts/L1/IL1ScrollMessenger.sol";
+
+/// @title ScrollValidator - makes cross chain call to update the Sequencer Uptime Feed on L2
+contract ScrollValidator is TypeAndVersionInterface, AggregatorValidatorInterface, SimpleWriteAccessController {
+  // solhint-disable-next-line chainlink-solidity/all-caps-constant-storage-variables
+  string public constant override typeAndVersion = "ScrollValidator 1.0.0";
+  int256 private constant ANSWER_SEQ_OFFLINE = 1;
+  uint32 private s_gasLimit;
```

Comment:
nit: I generally try to put the constants and immutable values together so it's clear what properties are packed together. There's no packing going on in here but I would still follow it by moving this below L2_UPTIME_FEED_ADDR